packages/@lwc/engine-core/src/framework/api.ts

```diff
@@ -560,6 +565,42 @@ function dc(
     return c(sel, Ctor, data, children);
 }
```

Comment:
Once [part 1](https://github.com/salesforce/lwc/pull/3337/commits/3c566fdcc0ffa12c1a0a813ec002f4eec52db2d6#diff-a284e53a17a2b48cc32e6bf3b121b3acb849a57df886d1cb92a2a46df5aebb7bR540-R563) is merged, this will be renamed the ddc (deprecated, `lwc:dynamic` syntax) function.  It is only listed here as the `dc` function to ensure the unit tests pass.